src/code-examples/go/connect-healthcheck/go.mod

```diff
@@ -2,7 +2,7 @@ module database-create
 
 go 1.18
 
-require github.com/codenotary/immudb v1.3.1
+require github.com/codenotary/immudb v1.4.0
```

Comment:
I'm not sure if this is needed - this sample, if used in 1.4.0 docs, will also be checked against 1.4.0 version.

Reply:
I think it's better to use the latest go sdk in the samples, even if we check them using older immudb servers

Probably we need to have version the samples as well